backend/api/invoices/delete.py

```diff
@@ -18,7 +18,8 @@ def delete_invoice(request: HtmxHttpRequest):
     try:
         invoice = Invoice.objects.get(id=delete_items.get("invoice", ""))
     except Invoice.DoesNotExist:
-        return JsonResponse({"message": "Invoice not found"}, status=404)
```

Comment:
The issue i'm finding with this is that HTMX will delete the row and not worry about the error. Whereas with a JsonResponse of 404 it will hold off the row deletion.

Reply:
But this case is covering situation when invoice doesn't exist so deleting it from html is what we need and the part where it's deleted from db is never reached (in case if it would be some not yet know bug and there would be such object in db).